src/main/java/com/conveyal/gtfs/GTFSCache.java

```diff
@@ -80,6 +59,51 @@ public GTFSCache(String bucket, String bucketFolder, File cacheDir) {
         this.bucketFolder = bucketFolder != null ? bucketFolder.replaceAll("\\/","") : null;
 
         this.cacheDir = cacheDir;
+        createCache(DEFAULT_CACHE_SIZE);
+    }
+
+    public GTFSCache(String bucket, String bucketFolder, File cacheDir, int cacheSize) {
+        if (bucket == null) LOG.info("No bucket specified; GTFS Cache will run locally");
+        else LOG.info("Using bucket {} for GTFS Cache", bucket);
+
+        this.bucket = bucket;
+        this.bucketFolder = bucketFolder != null ? bucketFolder.replaceAll("\\/","") : null;
+
+        this.cacheDir = cacheDir;
+        createCache(cacheSize);
+    }
+
+    /**
+     * Create cache of GTFSFeed objects with variable size.
+     * of
+     * @param cacheSize maximum number of feeds to be stored in cache
+     */
+    private void createCache(int cacheSize) {
```

Comment:
Rather than having a separate function to do this, I think it probably makes more sense to have a chained constructors, so you'd have:

GTFSCacche(bucket, cacheDir) which calls GTFSCache(bucket, bucketFolder, cacheDir) which calls GTFSCache(bucket, folder, cacheDir, cacheSize).